regridding/_regrid/_regrid_from_weights.py

```diff
@@ -52,11 +52,17 @@ def regrid_from_weights(
     :func:`regridding.regrid_from_weights`
     """
 
-    shape_input = np.broadcast_shapes(shape_input, values_input.shape)
-    values_input = np.broadcast_to(values_input, shape=shape_input, subok=True)
```

Comment:
Huh, so did this line get accidentally deleted? regardless we need to add `subok=True` to

Reply:
Looks like I just moved it below (line 64) that axis normilzation which needed the broadasted shape first.